cypress/support/step-definitions/carousel-steps.js

```diff
@@ -66,6 +74,10 @@ Then('previous button is not visible', () => {
   previousArrowButton().should('not.exist');
 });
 
+Then('previous button is disabled', () => {
+  previousArrowButton().should('be.disabled');
```

Comment:
needs to be added .and('have.attr', 'disabled')

Reply:
Added 👍